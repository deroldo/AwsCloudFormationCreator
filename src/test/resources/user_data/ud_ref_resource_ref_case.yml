MyData:
  Template: ref_resource_ref_case
  Foo: bar
  fooMap: foobar