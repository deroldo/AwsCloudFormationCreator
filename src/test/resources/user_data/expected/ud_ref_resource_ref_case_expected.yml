---
Resources:
  AnyResource1:
    Type: "AnyAwsType"
    Properties:
      Foo: "bar"
  OtherResource1:
    Type: "AnyAwsType"
    Condition: "AnyCondition1"
    DependsOn: "AnyResource1"
    Properties:
      Foo:
        Ref: "AnyResource1"
      Any:
        Fn::If:
        - "AnyCondition1"
        - "any2"
        - "bar"
      Other:
        Fn::GetAtt:
        - "AnyResource1"
        - "AnyThing"
      AnotherMap:
        Fn::FindInMap:
        - "anotherMap1"
        - "anotherMap"
        - "anotherMap"
      Map: "foobar"
Mappings:
  fooMap1:
    fooMap:
      fooMap:
        Foo: "test"
  anotherMap1:
    anotherMap:
      anotherMap:
        Foo: "test"
Conditions:
  AnyCondition1: true

